### ISSUE TITLE
[BUG] App hangs during "Parsing RTF"

### ISSUE THREAD

**Issue by Severgates** [Maradonna90/NewGAN-Manager#223]
**Describe the bug**

Identical configuration (graphic source files) worked in FM 23 with 1.3.1 version. 

Tried the same with 1.4.0 in FM24:

1. App run directly from CMD in Admin Mode (crashed without Admin Mode).
2. Configured like on the screen (set of faces worked properly in 1.3.1).
3. RTF file created correctly in game.
4. After clicking "Replaces faces" nothing happens (waited ca.10 minutes).

![image](https://github.com/Maradonna90/NewGAN-Manager/assets/157648404/5772220b-2ff6-4cb2-b6ce-3aae834e80ec)

Hint for those, who imported save from FM23 to FM24, your config file, created in 1.3.1 version works fine in FM24, just replace "graphics/pictures/person/" with "graphics/pictures/person/r-" (WordPad, CTRL+H, takes ages...) and paste new file to newgan graphics folder, dedicated for FM24.

**Comment by Strvsss** [comment]
Hi, have you found a fix for this? Same issue

**Comment by Severgates** [comment]
no, but FM Scout after latest update started to offer the same functionality, without necessity of exporting regens shortlist, so the answer what to do is pretty clear...

**Comment by Atti451** [comment]
Hi, i have the same issue, i did the correct things, and the first time everything was fine, after few season i wanted to do again, i clicked on preserve and nothing happened, i tryed to generate new faces with other face packs, but Still stuck on parsing rtf. Anyone have any solution? Thx.

**Comment by freopt** [comment]
> no, but FM Scout after latest update started to offer the same functionality, without necessity of exporting regens shortlist, so the answer what to do is pretty clear...

what FM scout?

**Comment by Severgates** [comment]
> > no, but FM Scout after latest update started to offer the same functionality, without necessity of exporting regens shortlist, so the answer what to do is pretty clear...
> 
> what FM scout?

another app

**Comment by freopt** [comment]
> > > no, but FM Scout after latest update started to offer the same functionality, without necessity of exporting regens shortlist, so the answer what to do is pretty clear...
> > 
> > 
> > what FM scout?
> 
> another app

link?

**Comment by Severgates** [comment]
> > > > no, but FM Scout after latest update started to offer the same functionality, without necessity of exporting regens shortlist, so the answer what to do is pretty clear...
> > > 
> > > 
> > > what FM scout?
> > 
> > 
> > another app
> 
> link?

You're 5 years old or been living under the rock for last 20 years? Use Google Search...
